docs/reference/index-modules.asciidoc

```diff
@@ -139,16 +139,19 @@ specific index module:
     The number of replicas each primary shard has.  Defaults to 1.
 
 `index.auto_expand_replicas`::
-
-    Auto-expand the number of replicas based on the number of data nodes in the cluster.
-    Set to a dash delimited lower and upper bound (e.g. `0-5`) or use `all`
-    for the upper bound (e.g. `0-all`). Defaults to `false` (i.e. disabled).
-    Note that the auto-expanded number of replicas only takes
-    <<shard-allocation-filtering,allocation filtering>> rules into account, but ignores
-    any other allocation rules such as <<shard-allocation-awareness,shard allocation awareness>>
-    and <<allocation-total-shards,total shards per node>>, and this can lead to the
-    cluster health becoming `YELLOW` if the applicable rules prevent all the replicas
-    from being allocated.
+Auto-expand the number of replicas based on the number of data nodes in the
```

Comment:
Why remove the indentation? Does this still display properly?

Reply:
Yep, see https://elasticsearch_69334.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/index-modules.html#dynamic-index-settings

I don't know if it's possible to add paragraph breaks (the `+` on its own line) to one of these blocks of text otherwise.